src/joystick/linux/SDL_sysjoystick.c

```diff
@@ -1153,6 +1271,43 @@ static void ConfigJoystick(SDL_Joystick *joystick, int fd)
         }
     }
 
+    /* Sensors are only available through the new unified event API */
+    if (fd_sensor >= 0 && (ioctl(fd_sensor, EVIOCGBIT(EV_ABS, sizeof(absbit)), absbit) >= 0)) {
+        if (test_bit(ABS_X, absbit) && test_bit(ABS_Y, absbit) && test_bit(ABS_Z, absbit)) {
+            for (i = 0; i < 3; ++i) {
+                struct input_absinfo absinfo;
+                if (ioctl(fd_sensor, EVIOCGABS(ABS_X + i), &absinfo) < 0) {
+                    break; /* do not report an accelerometer if we can't read all axes */
```

Comment:
`has_accelerometer` will be set to true even if we can't read all axes. This was in the original PR so I'll fix this.

Reply:
PR #8349 if you want to integrate this in here.

Reply:
> PR #8349 if you want to integrate this in here.

Thanks @meyraud705! Added in d4e4dd1edf954ebc35b651712c2a19662b6506e5